packages/gatsby-theme-carbon/gatsby-config.js

```diff
@@ -111,7 +117,7 @@ module.exports = (themeOptions) => {
             ...gatsbyRemarkPlugins,
           ],
           mdxOptions: {
-            remarkPlugins,
+            remarkPlugins: [...remarkPlugins, remarkGfm],
```

Comment:
```suggestion
            remarkPlugins: [remarkGfm, ...remarkPlugins],
```

I would flip these in case the user of the theme want to overwrite some settings or pass in their own version of the plugin they can do so.